lib/index.js

```diff
@@ -68,7 +68,7 @@ export const _dotProp = (string, object) => string
   .split('.')
   .reduce((acc, key) => acc[key], object);
 
```

Comment:
Could you add the prop parameter to the typings file? I think `(key: string, prop?: string)` should do it.